lib/internal/test_runner/test.js

```diff
@@ -137,6 +138,19 @@ function stopTest(timeout, signal) {
   return deferred.promise;
 }
 
+function testMatchesPattern(test, patterns) {
```

Comment:
nitpick perhaps this should move to utils

Reply:
I think it's okay here, because, while it is a helper function, it doesn't get used anywhere else, so I don't think there would be a reasons to modify another file. If you disagree, let me know and I'll move it.